src/worker/lodash-debounce.js

```diff
@@ -1,394 +1,390 @@
 // source: https://www.npmjs.com/package/lodash.debounce
 // version: 4.0.8
-// eslint-disable-next-line no-unused-vars
```

Comment:
this file should not be changed. If linter complains then it should be ignored.

Reply:
Except the required `export` statement for ES module instead of local `const` variable, there are only prettier fixes, as it is a part of our codebase. The removed `no-unused-vars` disabled rule does not apply, as we don't set that unused variable anymore (we use export). 

We can disable prettier for that file, but still the code has wrong indentation, so it should be fixed whatsoever ;)

And from another cotext, I would preffer to use `lodash-es` package, and take that implementation from that place (https://github.com/lodash/lodash/blob/master/debounce.js), rather than make a copy-paste solution.

Reply:
To minimize changes out of the scope of the main goal of this PR I added this file to .prettierignore. We can try to fix this issue (using it from the package) in another story.